### DISCUSSION TITLE
Games to try

### DISCUSSION THREAD

**Discussion by Joemama45** [skyline-emu/skyline#1468] [General]
Hey everyone, I'm here to test out games for skyline (for fun) to see if some work or some don't, leave some so I can test out! I will put these games in the compatibility list and status

**Comment by thekillerwave** [comment]
Dark Souls Remastered, XD

**Comment by Pokkoyori** [comment]
Zelda Breath Of The Wild :) And Mario Odyssey

**Comment by ghost** [comment]
Hollow Knight and Cuphead

**Comment by Mikatakhan** [comment]
Pathway
